usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -309,22 +301,67 @@ def handle(self, *args, **options):
         self.logger.info('Acquired award procurement data for ' + str(submission_id) + ', there are ' + str(len(procurement_data)) + ' rows.')
 
         for row in procurement_data:
-            # Yes, I could use Q objects here but for maintainability I broke it out
-            award = Award.objects.filter(piid=row['parent_award_id']).first()
-            if award is None:
-                award = Award.objects.filter(fain=row['parent_award_id']).first()
-            if award is None:
-                award = Award.objects.filter(uri=row['parent_award_id']).first()
-            if award is None:
-                self.logger.error('Could not find an award object with a matching identifier: ' + row['parent_award_id'])
-                continue
+            # Try and find the parent award
+            award = None
+            try:
+                award = find_or_create_summary_award(piid=row.get('piid'), fain=row.get('fain'), uri=row.get('uri'), parent_award_id=row.get('parent_award_id'))
```

Comment:
From what I understand, a summary award can only uniquely be identified _for sure_ by the combination of it's piid/uri + parent_award_id or fain. I.E. some agencies reuse piid/uri across different contract vehicles.